src/rest_utils.cpp

```diff
@@ -350,8 +354,11 @@ static Status parseOutputs(const ::KFSResponse& response_proto, rapidjson::Prett
         }
         size_t expectedElementsNumber = dataTypeSize > 0 ? expectedContentSize / dataTypeSize : 0;
 
-        if (!seekDataInValField && (tensor.datatype() != "BYTES" && response_proto.raw_output_contents(tensor_it).size() != expectedContentSize))
-            return StatusCode::REST_SERIALIZE_TENSOR_CONTENT_INVALID_SIZE;
+        if (!seekDataInValField && (tensor.datatype() != "BYTES" && response_proto.raw_output_contents(tensor_it).size() != expectedContentSize)) {
```

Comment:
is it related to metrics?

Reply:
It was useful for me to add it during debugging.